go.mod

```diff
@@ -1,20 +1,20 @@
 module github.com/brancz/kube-rbac-proxy
 
-go 1.23.4
+go 1.23.0
```

Comment:
Is this downgrade intentional?

Reply:
this thing indicates the minimal required go version feature-wise. :shrug: 

It doesn't change anything, but I can change it to 1.23.9